### PR TITLE
[FAB-17449] TC: add barebones test into daily suite

### PR DESCRIPTION
**Pull request by dongmingh**
This PR adds barebones test into daily test suite

Signed-off-by: Dongming Hwang <dongming@ibm.com>

### CODE REVIEW

**Comment by dongmingh** [comment]
@suryalnvs please review this PR.

File: regression/daily/runPteTestSuite.sh [hunk 1]
```diff
@@ -36,4 +47,24 @@ if [ ! -d "node_modules" ];then
 fi
 
 cd $DAILYDIR && ginkgo -v && echo "------> PTE tests completed"
+StatusPteNL=$?
+
+if [ $StatusPteNL == 0 ]; then
+    echo "------> PTE tests completed"
```
Comment (lindluni) [inline regression/daily/runPteTestSuite.sh]: We already echo this on line 49

File: regression/daily/runPteTestSuite.sh [hunk 2]
```diff
@@ -36,4 +47,24 @@ if [ ! -d "node_modules" ];then
...
+    echo "------> PTE tests failed with above errors"
+    exit 1
+fi
 archivePTE
+
+cd $BAREBONESDIR && ginkgo -v && echo "------> PTE barebones tests completed"
+StatusBarebones=$?
+
+if [ $StatusBarebones == 0 ]; then
+    echo "------> Barebones tests completed"
```
Comment (lindluni) [inline regression/daily/runPteTestSuite.sh:40]: Already echoed on line 60

File: regression/daily/runPteTestSuite.sh [hunk 3]
```diff
@@ -23,6 +24,16 @@ archivePTE() {
...
+archiveBarebones() {
+    if [ ! -z $GERRIT_BRANCH ] && [ ! -z $WORKSPACE ]; then
+        # GERRIT_BRANCH is a Jenkins parameter and WORKSPACE is a Jenkins directory.This function is used only when the test is run in Jenkins to archive the log files.
+        echo "------> Archiving generated logs"
+        tCurr=`date +%m%d`
+        mkdir -p $WORKSPACE/archives/PTE_Test_Logs
+        cp $FabricTestDir/regression/barebones/barebones-pteReport.log $WORKSPACE"/archives/PTE_Test_Logs/barebones-pteReport-"$tCurr".log"
+    fi
+}
+
```
Comment (lindluni) [inline regression/daily/runPteTestSuite.sh]: This won't ever be called, GERRIT_BRANCH doesn't exist in the current Jenkin's infrastructure, this was legacy to Hyperledgers old Jenkins.

File: regression/daily/runPteTestSuite.sh (hunk 1 above)
Comment (dongmingh) [reply regression/daily/runPteTestSuite.sh]: agree

Comment (dongmingh) [reply regression/daily/runPteTestSuite.sh]: agree

File: regression/daily/runPteTestSuite.sh (hunk 2 above)
Comment (dongmingh) [reply regression/daily/runPteTestSuite.sh:40]: agree

File: regression/daily/runPteTestSuite.sh (hunk 3 above)
Comment (dongmingh) [reply regression/daily/runPteTestSuite.sh]: will remove it

**Review by lindluni — APPROVED** [review]

**Pull request merged**